src/System.Diagnostics.Process/src/System/Diagnostics/Process.Linux.cs

```diff
@@ -57,6 +57,21 @@ internal DateTime StartTimeCore
             }
         }
 
+        /// <summary>Gets execution path</summary>
+        private string GetPathToOpenFile()
+        {
+            string[] allowedProgramsToRun = { "xdg-open", "gnome-open", s_kfmclient };
```

Comment:
>string[] allowedProgramsToRun = { "xdg-open", "gnome-open", s_kfmclient }; [](start = 12, length = 74)

Are these tools not installed on specific paths on Linux? if so, it would be better to check this path too.

The issue I am seeing is, this tool can be installed and not set as part of the environment path. at that time we'll fail even the tool is there.

Reply:
no the tools are not installed by default. When they are installed the get located in the path.

Reply:
Is it possible the tools can get installed and not be in the PATH? do the tools always exist in a specific path?

Reply:
From what I checked, for all three, the location they get installed, is available in path:

They get installed in /usr/bin. And this directory is already in the path.

I tried Ubuntu 14.04 for this experiment.

Reply:
ok, sounds good then.